MatrixSDK/Data/MXRoomSummaryDataTypes.h

```diff
@@ -26,7 +26,10 @@ typedef NS_OPTIONS(NSInteger, MXRoomSummaryDataTypes)
     MXRoomSummaryDataTypesServerNotice = 1 << 4,
     MXRoomSummaryDataTypesHidden = 1 << 5,
     MXRoomSummaryDataTypesSpace = 1 << 6,
-    MXRoomSummaryDataTypesConferenceUser = 1 << 7
+    MXRoomSummaryDataTypesConferenceUser = 1 << 7,
+    MXRoomSummaryDataTypesNotDirect = 1 << 8,
+    MXRoomSummaryDataTypesUnread = 1 << 9,
+    MXRoomSummaryDataTypesRead = 1 << 10,
```

Comment:
It feels a bit odd to have `Direct`/`NotDirect` & `Unread`/`Read` in here when the omission of one implies the other, but I haven't had a look at the Element PR yet to see the full context.

Reply:
Ok, I see the use now. Logically my brain would have gone for adding `Direct` to `notDataTypes` instead of `NotDirect` but the way this works in the `RecentsListService` is much more readable 🙂.

Reply:
I've removed unnecessary `MXRoomSummaryDataTypesNotDirect` and `MXRoomSummaryDataTypesRead ` types